cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -176,6 +177,12 @@ func (m command) build() *cli.Command {
 			Usage:       "Specify a pattern the CSV mount specifications.",
 			Destination: &opts.csv.ignorePatterns,
 		},
+		&cli.StringSliceFlag{
+			Name:        "disable-hook",
```

Comment:
Should we also add a disable-all-hooks flag so people don't have to enumerate all of them to disable them.

Also, should we add a list-hooks command so people can discover what hooks are available for disabling

Reply:
@klueska my thoughts on how to handle the disabling of all hooks is to also accept `all` as a valid hook name. So that `--disable-hook(s)=all` can be specified.

Reply:
> Also, should we add a list-hooks command so people can discover what hooks are available for disabling

Do you mean something like:
```
nvidia-ctk cdi list-hooks
```?

Reply:
I have added an alias of `disable-hooks` and a better Usage description.